src/react/components/_academics/AcademicSummary/PrimarySection.js

```diff
@@ -33,10 +33,14 @@ let SingleSection = ({
         grading={grading}
         units={units}
         canViewGrades={canViewGrades}
+        showPoints={showPoints}
       />
     );
   }
 
+
+  let formattedlawUnits = lawUnits ? lawUnits : '0.0'
```

Comment:
@frankiepun - Same here. Also I would make the variable `formattedLawUnits`, instead of `formattedlawUnits`.

Reply:
I updated the code